distmap

```diff
@@ -14,11 +15,12 @@ use HadoopTrimming;
 use HadoopMapping;
 use DataDownload;
 use DataMerge;
+use DataDownloadAndMerge;
 use DownloadTrimmedRead;
 use DataCleanup;
 
 my $script = "distmap";
-my $version = "2.7.1";
+my $version = "1.1.0"; # use semantic versioning + define the v2.7.1 API as 1.0.0
```

Comment:
If you want to use semantic versioning, I will rename the current version (a different PR getting in before this one, maybe with the cleanup) to be `3.0.0-alpha` and start with SemVer from there onwards (e.g., this PR might be `3.0.1-alpha`. Because you are not planing to add a change in the API, this should not change to version 4.0.0 regarding SemVer until new arguments are coming in, such as sort order and so on.

In addition, keeping `alpha` in the name will prevent people for using it until the cleanup and `ReadTools` integration is in.

Reply:
ok

Reply:
That's already modified in #10 - a rebase may solve the conflicting files.